src/components/switch/__tests__/__snapshots__/Switch.spec.js.snap

```diff
@@ -25,7 +25,7 @@ exports[`CdrSwitch > matches snapshot 1`] = `
   >
     <svg
       aria-hidden="true"
-      class="cdr-icon cdr-icon--medium cdr-switch__button-icon"
+      class="cdr-icon cdr-switch__button-icon"
```

Comment:
An example where an invalid size class was being applied only in the test environments